bcipy/helpers/convert.py

```diff
@@ -167,23 +285,103 @@ def write_edf(output_path: str,
             channel_info.append(ch_dict)
             data_list.append(raw_data[i])
 
+        if events:
+            writer.set_number_of_annotation_signals(annotation_channels)
+            for onset, duration, label in events:
+                writer.writeAnnotation(onset, duration, label)
+
         writer.setSignalHeaders(channel_info)
         writer.writeSamples(data_list)
 
+    except Exception as error:
+        logger.info(error)
+        raise error
+    finally:
+        writer.close()
+    return output_path
+
+
+def write_bdf(output_path: str,
```

Comment:
There is a lot of redundancy between this method and `write_edf`. If we exclude the digital_min and digital_max (see other comment), the only difference is the file_type. Maybe that can be a parameter and we can have a single method with a default value of `FILETYPE_EDFPLUS`